src/sagemaker/tensorflow/README.rst

```diff
@@ -560,99 +207,82 @@ After attaching, the estimator can be deployed as usual.
-'/', in end of the url). If TensorBoard started on a different port, adjust these URLs to match.
-
-Note that TensorBoard is not supported when passing wait=False to ``fit``.
-
-
-Deploying TensorFlow Serving models
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-After a TensorFlow estimator has been fit, it saves a TensorFlow SavedModel in
-the S3 location defined by ``output_path``. You can call ``deploy`` on a TensorFlow
-estimator to create a SageMaker Endpoint.
-
-SageMaker provides two different options for deploying TensorFlow models to a SageMaker
-Endpoint:
-
-- The first option uses a Python-based server that allows you to specify your own custom
-  input and output handling functions in a Python script. This is the default option.
-
-  See `Deploying to Python-based Endpoints <deploying_python.rst>`_ to learn how to use this option.
-
-
-- The second option uses a TensorFlow Serving-based server to provide a super-set of the
-  `TensorFlow Serving REST API <https://www.tensorflow.org/serving/api_rest>`_. This option
-  does not require (or allow) a custom python script.
-
-  See `Deploying to TensorFlow Serving Endpoints <deploying_tensorflow_serving.rst>`_ to learn how to use this option.
+The ``TensorFlow`` constructor takes both required and optional arguments.
 
+Required:
+
+- ``entry_point (str)`` Path (absolute or relative) to the Python file which
+  should be executed as the entry point to training.
+- ``role (str)`` An AWS IAM role (either name or full ARN). The Amazon
+  SageMaker training jobs and APIs that create Amazon SageMaker
+  endpoints use this role to access training data and model artifacts.
+  After the endpoint is created, the inference code might use the IAM
+  role, if accessing AWS resource.
+- ``train_instance_count (int)`` Number of Amazon EC2 instances to use for
+  training.
+- ``train_instance_type (str)`` Type of EC2 instance to use for training, for
+  example, 'ml.c4.xlarge'.
+
+Optional:
+
+- ``source_dir (str)`` Path (absolute or relative) to a directory with any
+  other training source code dependencies including the entry point
+  file. Structure within this directory will be preserved when training
+  on SageMaker.
+- ``dependencies (list[str])`` A list of paths to directories (absolute or relative) with
+  any additional libraries that will be exported to the container (default: ``[]``).
+  The library folders will be copied to SageMaker in the same folder where the entrypoint is copied.
+  If the ```source_dir``` points to S3, code will be uploaded and the S3 location will be used
+  instead. Example:
+
+      The following call
+      >>> TensorFlow(entry_point='train.py', dependencies=['my/libs/common', 'virtual-env'])
+      results in the following inside the container:
+
+      >>> $ ls
+
+      >>> opt/ml/code
+      >>>     ├── train.py
+      >>>     ├── common
+      >>>     └── virtual-env
```

Comment:
actually, maybe this section shouldn't be extra indented - it's showing up as quoted text.